.github/workflows/check-deps.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         with:
           ref: ${{ github.head_ref }}
       - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v3
+        uses: actions/setup-python@0ebf233433c08fb9061af664d501c3f3ff0e9e20 # v3
```

Comment:
i think remove the v3 from the end as it will not get updated

Reply:
Done.